Bio/SeqUtils/ProtParam.py

```diff
@@ -125,7 +125,7 @@ def get_amino_acids_percent(self):
 
     def molecular_weight(self):
         """Calculate MW from Protein sequence."""
-        return molecular_weight(self.sequence, monoisotopic=self.monoisotopic)
+        return molecular_weight(self.sequence, seq_type="protein", monoisotopic=self.monoisotopic)
```

Comment:
Adding a new argument it should be last by default so that existing calls to ``.molecular_weight(

Reply:
Not sure how that got posted, I didn't finish writing it and thought I deleted it - the function signature didn't change.